.gitignore

```diff
@@ -194,7 +194,6 @@ frontend/playground-frontend/src/backend_outputs/*
 
 # Terraform Files
 dlp-terraform/.terraform
-dlp-terraform/outputs
```

Comment:
@thomaschin35 you're sure this is correct

Reply:
Yes this is correct